ansible/roles/deploy-plugins/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+s3_bucket_name: {{aws_s3_bucket | d('')}}
```

Comment:
we don't have to override this vavriable, right ? we can use the same `aws_s3_bucket`

Reply:
if there is no default value, we will get variable undefined error for azure @rjshrjndrn